sdk/src/lib.rs

```diff
@@ -30,8 +31,10 @@ pub mod env;
 pub mod store;
 
 /// Transaction Management Module
+#[cfg(any(feature = "std", feature = "wasm"))]
 pub mod tx;
 
+#[cfg(feature = "std")]
```

Comment:
This should be possible to be run in `no_std`? Therefore we should just have the `tangle_subxt/std` added to `std` feature in the TOML. If something needs to change on the package itself please handle it.